be/src/common/config.h

```diff
@@ -680,7 +680,7 @@ CONF_Int64(download_cache_buffer_size, "10485760");
 // so if there are too many segment in a rowset, the compaction process
 // will run out of memory.
 // When doing compaction, each segment may take at least 1MB buffer.
-CONF_mInt32(max_segment_num_per_rowset, "200");
+CONF_mInt32(max_segment_num_per_rowset, "1500");
```

Comment:
I' m not sure if there are too many segments in a rowset, the query operation will cost too much memory?